core/src/main/java/org/bitcoinj/core/Wallet.java

```diff
@@ -746,6 +746,22 @@ public DeterministicKey getWatchingKey() {
     }
 
     /**
+     * Returns true if this wallet has watch-only keys only, and false if it has non-watch-only keys only.
```

Comment:
This is a bit awkwardly phrased. How about:

"Returns whether this wallet consists entirely of watching keys (unencrypted keys with no private part). Mixed wallets are forbidden."